### DISCUSSION TITLE
Is there a date for version 0.11.0 release?

### DISCUSSION THREAD

**Discussion by sneko** [hbldh/bleak#485] [Resolved discussions] [answered]
Hi,

I saw a few recent interesting commits I would like to benefit from. For now I'm getting the `develop` branch but it's a bit complicated to deal with, mainly due to package managers.

I just wanted to know if you have a date for the next release @hbldh ?

Thank you,

**Comment by hbldh** [comment]
Yes. A release is planned. It just has so happened that a lot of long standing changes were implemented, review and tested for this release, so it has taken some time to get things done. It is not far off though.

**Comment by hbldh** [comment, marked as answer]
Version 0.11.0 has now been released.
